src/guest_webapp/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+# not sure which version to use
+version: '3.8'
+
+services:
+  react-frontend:
+    build:
+      context: .
+      # dockerfile: Dockerfile
+    ports:
+      - "5173:5173"
+    volumes:
+      - .:/guest_webapp
+      - /guest_webapp/node_modules
+    command: npm run dev -- --host 0.0.0.0 # host 0.0.0.0 works on all interfaces
```

Comment:
NOT ACTIONABLE:
Comments like these are good for learning, in general tho, it's cleaner not to have them. This is something to talk about in our meeting.